pymemcache/client.py

```diff
@@ -999,6 +1009,256 @@ def __delitem__(self, key):
         self.delete(key, noreply=True)
 
 
+class ShardingClient(object):
+
+    def __init__(self, clients, black_list_timeout=30):
```

Comment:
Please document this method similarly to the constructors for PooledClient and Client.